.claude/skills/admin-ui/SKILL.md

```
@@ -71,3 +74,30 @@ agent: general-purpose
 ### HTML
 - 스크립트 순서: `auth.js` → `api.js` → `pagination.js` → `{page}.js`
 - 시맨틱 HTML5 + 접근성 속성 (aria-label, role)
+
+## Examples
+
+### Example 1: 호스트 관리 목록 페이지
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)